### PR TITLE
feat: add speechToText feature

### PR DESCRIPTION
**Pull request by mss8087**
### What is the issue this pull request related with?

resolved #<issue-number> 

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Code builds correctly
- [x] Created tests which fail without the change (if possible)
- [x] All tests passing
- [x] Extended the README / documentation, if necessary

### Description
Please describe your pull request.

💔Thank you!

### CODE REVIEW

**Comment by zeroFruit** [comment]
### TODO

Need to integration test

File: backend/api_gateway/group_endpoint/views.py [hunk 1]
```diff
@@ -21,10 +21,10 @@ def with_json_response(status, data):
 def group(request):
     if request.method == 'POST':
         return __create_group(request)
-    elif request.method == 'PUT':
-        return __update_group(request)
+    #elif request.method == 'PUT':
+    #    return __update_group(request)
```
Comment (zeroFruit) [inline backend/api_gateway/group_endpoint/views.py]: Please remove comment block

File: frontend/src/containers/RequestCallContainer/RequestCallContainer.js [hunk 2]
```diff
@@ -1,30 +1,34 @@
...
   }));
 
   useEffect(() => {
-    // TODO: replace id with user id
-    const stream = grpcClient.createGrpcStream({ id: 3 });
+    const stream = grpcClient.createGrpcStream({ id: 11/*user.id*/ });
     stream.on('data', message => {
       const parsed = JSON.parse(message.getData());
-      if (parsed._type_name !== USER_CREATED_EVENT) {
+      //console.log(parsed);
```
Comment (zeroFruit) [inline frontend/src/containers/RequestCallContainer/RequestCallContainer.js]: please remove comment block

File: frontend/src/containers/RequestCallContainer/RequestCallContainer.js [hunk 3]
```diff
@@ -37,14 +41,42 @@ function RequestCallContainer({ history }) {
...
+    return (
+      <RequestCallSection
+        driverId={user.driverId}
+        groupId={group.groupId}
+        onClickRequestCall={onClickRequestCall}
+      />
+    );
+  }
+
+*/
```
Comment (zeroFruit) [inline frontend/src/containers/RequestCallContainer/RequestCallContainer.js]: please remove comment outed code

File: frontend/src/containers/RequestCallContainer/RequestCallContainer.test.js [hunk 4]
```diff
@@ -127,4 +127,5 @@ describe('<RequestCallContainer />', () => {
 
     expect(getByText('MockGroupPage')).toHaveTextContent('MockGroupPage');
   });
+  */
```
Comment (zeroFruit) [inline frontend/src/containers/RequestCallContainer/RequestCallContainer.test.js]: please remove commented code

File: frontend/src/lib/speech/speech.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import dotenv from "dotenv"
```
Comment (zeroFruit) [inline frontend/src/lib/speech/speech.js]: please rename this directory into `lib/azure/speech.js`

File: frontend/src/containers/RequestCallContainer/RequestCallContainer.js [hunk 6]
```diff
@@ -37,14 +40,42 @@ function RequestCallContainer({ history }) {
...
+  /*
+  // for test
+ return (
+  <RequestCallSection
+    driverId={6}
+    groupId={10}
+    onClickRequestCall={onClickRequestCall}
+  />
+  );
+  */
```
Comment (zeroFruit) [inline frontend/src/containers/RequestCallContainer/RequestCallContainer.js]: please remove commented block

**Comment by mss8087** [comment]
TODO: api_gateway/group_endpoint/views.py test

**Pull request merged**